ios/FluentUI/Other Cells/ActionsCell.swift

```diff
@@ -15,51 +15,85 @@ import UIKit
  `topSeparatorType` and `bottomSeparatorType` can be used to show custom horizontal separators. Make sure to remove the `UITableViewCell` built-in separator by setting `separatorStyle = .none` on your table view.
  */
 @objc(MSFActionsCell)
-open class ActionsCell: UITableViewCell {
+open class ActionsCell: UITableViewCell, TokenizedControlInternal {
     @objc(MSFActionsCellActionType)
     public enum ActionType: Int {
         case regular
         case destructive
         case communication
 
-        func highlightedTextColor(for window: UIWindow) -> UIColor {
+        func highlightedTextColor(tokens: ActionsCellTokens) -> UIColor {
             switch self {
             case .regular:
-                return Colors.primary(for: window).withAlphaComponent(0.4)
+                return UIColor(dynamicColor: tokens.regularTextColor).withAlphaComponent(0.4)
             case .destructive:
-                return Colors.Table.ActionCell.textDestructiveHighlighted
+                return UIColor(dynamicColor: tokens.destructiveTextColor).withAlphaComponent(0.4)
```

Comment:
why do we have alpha values all the sudden?

Reply:
oh i guess it used to be alpha?

Reply:
Yep!